values.yaml

```diff
@@ -279,26 +306,69 @@ nifi:
   persistence:
     enabled: true
   properties:
-    externalSecure: false
-    isNode: true
-    httpPort: 8080
-    httpsPort: null
-    webProxyHost: # set it by the same host than nifi.traefikIngress.host if traefik ingress is enabled
+    sensitiveKey: changeMechangeMe # Must have at least 12 characters
+    isNode: false
+    httpsPort: 8443
+    webProxyHost: <your_nifi_domain_name> # set it by the same host than nifi.traefikIngress.host if traefik ingress is enabled
     clusterPort: 6007
-    clusterSecure: false
+    customLibPath: "/opt/configuration_resources/custom_lib"
   auth:
+    admin: CN=admin, OU=NIFI
+    SSL:
+      keystorePasswd: changeMe
+      truststorePasswd: changeMe
+  
+  # Automaticaly disabled if OIDC or LDAP enabled 
+    singleUser:
+      username: username
+      password: changemechangeme # Must to have at least 12 characters
+
     ldap:
-      enabled: true
+      enabled: false
       host: #ldap://<hostname>:<port>
-      searchBase: #CN=admin,DC=ldap,DC=example,DC=be
+      searchBase: #CN=Users,DC=ldap,DC=example,DC=be
       admin: #cn=admin,dc=ldap,dc=example,dc=be
       pass: #ChangeMe
       searchFilter: (objectClass=*)
       userIdentityAttribute: cn
+      authStrategy: SIMPLE # How the connection to the LDAP server is authenticated. Possible values are ANONYMOUS, SIMPLE, LDAPS, or START_TLS.
+      identityStrategy: USE_DN
+      authExpiration: 12 hours
+    oidc:
+      enabled: false
+      discoveryUrl: https://login.microsoftonline.com/<your_tenant_id>/v2.0/.well-known/openid-configuration
+      clientId: <your_application_id>
+      clientSecret: <your_client_secret>
+      claimIdentifyingUser: email
+      admin: <your_admin_email>
+      ## Request additional scopes, for example profile
+      additionalScopes: 
+  registry:
+    enabled: false
+  # ---- add new processors for InfluxDB ---
+  extraVolumeMounts:
+    - name: mycustomlibs
+      mountPath: /opt/configuration_resources/custom_lib
+  extraVolumes: # this will create the volume from the directory
+    - name: mycustomlibs
+      emptyDir: {}
+  initContainers: 
+    install-processor:
+      image: "ellerbrock/alpine-bash-curl-ssl"
+      imagePullPolicy: "IfNotPresent"
+      command:
+        - bash
+        - -c
+        - |
+          cd /data
+          wget https://github.com/influxdata/nifi-influxdb-bundle/releases/download/v1.13.0/nifi-influx-database-nar-1.13.0.nar
```

Comment:
The whole custom lib thing should probably not be here in the default values.yaml

Reply:
What I meant is that we should not be downloading influxdb processor in this default values.yaml file.
We can comment it out so that it provides an example on how to do that though.